pallets/rc-migrator/src/staking/message.rs

```diff
@@ -445,21 +445,57 @@ impl<T: pallet_staking::Config> StakingMigrator<T> {
 		use pallet_staking::*;
 
 		StakingValues {
-			validator_count: ValidatorCount::<T>::take(),
-			min_validator_count: MinimumValidatorCount::<T>::take(),
-			min_nominator_bond: MinNominatorBond::<T>::take(),
-			min_validator_bond: MinValidatorBond::<T>::take(),
-			min_active_stake: MinimumActiveStake::<T>::take(),
-			min_commission: MinCommission::<T>::take(),
+			validator_count: if ValidatorCount::<T>::exists() {
+				Some(ValidatorCount::<T>::take())
+			} else {
+				None
+			},
```

Comment:
```suggestion
			validator_count: ValidatorCount::<T>::exists().then(ValidatorCount::<T>::take),
```
A bit shorter, but its fine like this as well 😋